Add blog post about the mainframe priorities survey (#6072)
open-telemetry/opentelemetry.io: content/en/blog/2025/otel-mainframe-priorities-survey.md

---
title: OpenTelemetry on Mainframe Priorities Survey
linkTitle: Mainframe Priorities Survey

> date: 2025-01-24

Reviewer: ```suggestion
date: 2025-01-30
```

let's keep this open until we set the right date.

Reviewer: @rrschulze let's keep this open until we decide on a final schedule, might push out to tomorrow, so I want to ensure that we have the right date set.

prioritize and implement targeted activities to accelerate the adoption of
OpenTelemetry on the mainframe platform. Such activities could be the expedited
porting of selected SDKs for code instrumentation to z/OS or the definition of
Semantic Conventions for specific aspects of the platform. Once the survey is
complete, we will publish the results and a prioritized list of activities on
the websites of the OpenTelemetry project and The Open Mainframe Project.

## Taking the survey

The [survey](https://www.surveymonkey.com/r/HGTD2KJ) is structured in two
sections. The first section gathers input about your role and background. The
second section gathers the priorities of your organization for enabling
OpenTelemetry on mainframes. Responding to the 20 questions will take
approximately 15 minutes. We appreciate your time spent on the survey and look
forward to receiving your feedback!

We invite you to take the [survey](https://www.surveymonkey.com/r/HGTD2KJ)! To
learn more about the work of the OpenTelemetry on Mainframe SIG, join the
conversation at
[#otel-mainframes](https://cloud-native.slack.com/archives/C05PXDFTCPJ) on

> [CNCF Slack](https://slack.cncf.io/).

Reviewer 2: ```suggestion
[CNCF Slack](https://slack.cncf.io/).

_A version of this article was [originally posted] on the Open Mainframe Project blog.

[originally posted]: {{% param canonical_url %}}

[survey]: https://www.surveymonkey.com/r/HGTD2KJ

```

of the OpenTelemetry project has made major efforts to provide specifications
and implementations that unify the collection, processing and distribution of
telemetry.

Given the breadth of the OpenTelemetry project and sophisticated capabilities of
the mainframe, we as contributors to the SIG want to better understand the
priorities for making the best use of OpenTelemetry on the mainframes. We seek
feedback from a broad range of mainframe users and users of observability
solutions, developers, site reliability engineers, application owners,
architects, DevOps engineers and all other experts in the field.

The results of the [survey](https://www.surveymonkey.com/r/HGTD2KJ) will help us
prioritize and implement targeted activities to accelerate the adoption of
OpenTelemetry on the mainframe platform. Such activities could be the expedited
porting of selected SDKs for code instrumentation to z/OS or the definition of
Semantic Conventions for specific aspects of the platform. Once the survey is
complete, we will publish the results and a prioritized list of activities on
the websites of the OpenTelemetry project and The Open Mainframe Project.

> Taking the survey
> -----------------

Reviewer 3: ```suggestion
## Taking the survey
```

The [survey](https://www.surveymonkey.com/r/HGTD2KJ) is structured in two
sections. The first section gathers input about your role and background. The
second section gathers the priorities of your organization for enabling
OpenTelemetry on mainframes. Responding to the 20 questions will take
approximately 15 minutes. We appreciate your time spent on the survey and look

> forward to receiving your feedback! **Take the survey
> [here](https://www.surveymonkey.com/r/HGTD2KJ)!** Learn more about the work of
> the OpenTelemetry on Mainframe SIG by joining the conversation on Slack
> [here](https://cloud-native.slack.com/archives/C05PXDFTCPJ).

Reviewer 3: Better to use more descriptive text than 'here' for links.

```suggestion
forward to receiving your feedback! 

We invite you to take the [survey](https://www.surveymonkey.com/r/HGTD2KJ)! To learn more about the work of the OpenTelemetry on Mainframe SIG, join the conversation at [#otel-mainframes](https://cloud-native.slack.com/archives/C05PXDFTCPJ) on [CNCF Slack](https://slack.cncf.io/).
```

--
title: OpenTelemetry on Mainframe Priorities Survey

> linkTitle: Mainframe Priorities Survey # Mandatory, make sure that your short title.
> date: 2025-01-23
> author: >-
> [Ruediger Schulze](https://github.com/rrschulze) (IBM)
> canonical_url: <https://openmainframeproject.org/blog/opentelemetry-on-mainframe-priorities-survey/?utm_content=322043016&utm_medium=social&utm_source=linkedin&hss_channel=lcp-35545633>
> issue: [#6045](https://github.com/open-telemetry/opentelemetry.io/issues/6045)

Reviewer 2: ```suggestion
linkTitle: Mainframe Priorities Survey 
date: 2025-01-24
author: '[Ruediger Schulze](https://github.com/rrschulze) (IBM)'
canonical_url: https://openmainframeproject.org/blog/opentelemetry-on-mainframe-priorities-survey/
issue: https://github.com/open-telemetry/opentelemetry.io/issues/6045
```

---
title: OpenTelemetry on Mainframe Priorities Survey
linkTitle: Mainframe Priorities Survey

> date: 2025-01-30

Reviewer: ```suggestion
date: 2025-01-31
```

The results of the [survey] will help us prioritize and implement targeted
activities to accelerate the adoption of OpenTelemetry on the mainframe
platform. Such activities could be the expedited porting of selected SDKs for
code instrumentation to z/OS or the definition of Semantic Conventions for
specific aspects of the platform. Once the survey is complete, we will publish
the results and a prioritized list of activities on the websites of the
OpenTelemetry project and The Open Mainframe Project.

## Taking the survey

The [survey] is structured in two sections. The first section gathers input
about your role and background. The second section gathers the priorities of
your organization for enabling OpenTelemetry on mainframes. Responding to the 20
questions will take approximately 15 minutes. We appreciate your time spent on
the survey and look forward to receiving your feedback!

We invite you to take the [survey]. To learn more about the work of the
OpenTelemetry on Mainframe SIG, join the conversation at
[#otel-mainframes](https://cloud-native.slack.com/archives/C05PXDFTCPJ) on
[CNCF Slack](https://slack.cncf.io/).

> \_A version of this article was [originally posted] on the Open Mainframe
> Project blog.

Reviewer 2: ```suggestion
_A version of this article was [originally posted] on the Open Mainframe
Project blog._
```
